### DISCUSSION TITLE
Cannot adjust image slider buttons.

### DISCUSSION THREAD

**Discussion by IvayloDaskalov** [tailwindlabs/tailwindcss#15435] [Help]
Hello guys, 
For a website I am trying to create an image slider, where a person can slide between images with arowheads. For that i have the following code: 

//<div id="menu" class="parallax-window" data-parallax="scroll" data-image-src="img/antique-cafe-bg-02.jpg">
        <div id="menu-items-pad" class="container mx-auto tm-container flex flex-col items-center justify-center">
            <div class="text-center mb-8">
                <h2 id="menu-title" class="bg-white tm-text-brown py-6 px-12 text-4xl font-medium inline-block rounded-md">Our Menu</h2>
            </div>
            <div class="bg-red-600 p-10 mb-5 relative">
                <div id="menu-slider" class="relative h-full flex justify-center items-center overflow-hidden">
                    <img id="menu-page-1" src="img/menu-page-1.png" alt="Menu Page 1" class="rounded-lg hidden max-h-full object-contain w-full">
                    <img id="menu-page-2" src="img/menu-page-2.png" alt="Menu Page 2" class="rounded-lg hidden max-h-full object-contain w-full">
                    <img id="menu-page-3" src="img/menu-page-3.png" alt="Menu Page 3" class="rounded-lg hidden max-h-full object-contain w-full">
                    <img id="menu-page-4" src="img/menu-page-4.png" alt="Menu Page 4" class="rounded-lg hidden max-h-full object-contain w-full">
                </div>
                <button id="prev-btn" class="absolute left-4 top-1/2 transform -translate-y-1/2 bg-transparent text-white border-2 border-transparent hover:border-black p-2 rounded-full hover:bg-black transition">
                    &#8592;
                </button>
                <button id="next-btn" class="absolute right-4 top-1/2 transform -translate-y-1/2 bg-transparent text-white border-2 border-transparent hover:border-black p-2 rounded-full hover:bg-black transition">
                    &#8594;
                </button>
            </div>
        </div>
  //  </div>

//<script>
    const slider = document.getElementById('menu-slider');
    const images = slider.querySelectorAll('img');
    const prevBtn = document.getElementById('prev-btn');
    const nextBtn = document.getElementById('next-btn');
    let currentIndex = 0;
    function updateSlider() {
        images.forEach((img, index) => {
            img.classList.toggle('hidden', index !== currentIndex);
        });
    }
    prevBtn.addEventListener('click', () => {
        currentIndex = (currentIndex - 1 + images.length) % images.length;
        updateSlider();
    });

    nextBtn.addEventListener('click', () => {
        currentIndex = (currentIndex + 1) % images.length;
        updateSlider();
    });
    updateSlider();
//</script>

However my arrows apear on top of each other in the left lower corner of the images. Not to mention that i struggle to find a way to put a container/wrapper around the the images themselfs, because i think there are a bit too big right now. Can you help me fix the previuos/ next arrowheads and maybe wrap the image slider?

**Comment by wongjn** [comment]
Hi! Could you look at formatting your code in a code block please?
